apps/core-angular-cli/package.json

```diff
@@ -22,7 +22,7 @@
     "@angular/platform-browser-dynamic": "~10.0.3",
     "@angular/router": "~10.0.3",
     "@clr/city": "^1.1.0",
-    "@clr/core": "./packages/core/dist/core",
+    "@clr/core": "latest",
```

Comment:
Just to double check, does the build still properly swap and set the package versions in the final dist output for publication?

Reply:
Yea, you can verify that by seeing the package.json version inside `node_modules` folder of the target package.